Week1/homework/nadin.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+let greetingIndonesian = 'Ciao';
+let greetingArabic = 'مرحبا ';
+let greetingSpanish = 'Halo';
+ console.log(greetingIndonesian, greetingArabic, greetingSpanish);
+
+
+console.log("I'm awesome");
+
+
+let x;
+console.log("The value of my variable number will be: undefined");
+console.log(x);
+x = 1;
+console.log("The value of my variable number will be: 1");
+console.log(x);
+
+let y = "the first string";
+console.log("the value of my variable y will be: the first string");
+console.log(y);
+y = "the second string";
+console.log("the value of my variable string1 will be: the second string");
+console.log(y);
+
+console.log(Math.round(7.25));
+// expected output: 8
+
+let z = 7.25;
+console.log(z);
+
+let a = 8;
+console.log(a);
+
+console.log(z !== a);
+console.log(z == a);
+console.log(z <= a);
```

Comment:
Hi Nadin, I think you perhaps mis-understood the requirement here. We want to do this:
`5.3 Declare another variable a that has the value of z but rounded to the nearest integer.`
`5.5 So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.`.

That means, instead of hardcoding `let a = 8;`, you need to do this `const a = Math.round(z);`.

After this, you need to compare the two variables `a` and `z` and store the maximum of it in a new variable `b`. For that, I would like you to explore two ways: using `if-else` block or using `Math.max` method. 🙂

Reply:
Thanks a lot! You are right, I mis-understood the requirement, that gave me the chance to read about Math.max , Math.min and I use it with if/if-else. ;) It's done!